_gsocproposals/2020/proposal_CMSSW-ROOT-Mesh-Allocator.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Enable the Mesh Memory Allocator in ROOT
+layout: gsoc_proposal
+project:
+  - CMSSW
```

Comment:
I don't think we support a proposal to belong to 2 projects, it has to be only one. It can only involve multiple organizations.